pyformance/reporters/influx.py

```diff
@@ -87,10 +105,14 @@ def _create_database(self):
     def report_now(self, registry=None, timestamp=None):
         if self.autocreate_database and not self._did_create_database:
             self._create_database()
-        timestamp = timestamp or int(round(self.clock.time()))
```

Comment:
Note that the old code was rounding the time (same for events time at line 127). I thought that is an unexpected behavior so I removed the rounding part, in other words, for "seconds" precision for example, the code drops the milliseconds data.